frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -446,10 +446,12 @@ void main(List<String> args) async {
     expect(cnt, 5);
   });
 
+  /*
   test('Throw CustomError', () async {
     expect(() async => await api.returnErrCustomError(), throwsA(isA<CustomError>()));
   });
-
+  */
```

Comment:
(Dummy review comment here. Resolve this comment only after all tests are re-enabled, such that GitHub auto check and ensure we do not accidentially merge before enabling all tests)